### PR TITLE
Added language auto-detection for voice being cloned

### LINKED ISSUE #75: Auto-detect source audio language
The first 10 seconds could be clipped (if the source audio is longer than 10 seconds) to auto-detect language of source audio.

This would allow us to remove another input from the UI and CLI.

### PR DESCRIPTION
**Pull request by andrew-fennell**
## Summary
Input audio file for the **voice to be cloned** and it automatically decides which encoder to use.

This removes one more user input by taking the first 10 seconds of the audio clip, and decided which language is being spoken. This uses the auto-detection already setup in the ```backend.modules.STT``` module.

I used the same library that @JacobSmith1020 used (in #52) to clip the first 10 seconds of the audio, IF the audio is longer than 10 seconds.

## Example
image

Closes #75

### CODE REVIEW

**Comment by andrew-fennell** [comment]
I also removed duplicates of languages for now from ```backend/modules/languages.py```.

I think this is in the project's best interest (for now), as we won't be supporting all lingual regions of each language as of right now.

**Review by JacobSmith1020 — CHANGES_REQUESTED** [review]
I ran this with the short BillMaher22.wav file and the TCOC40.wav file and both fail with an index out of range error for ```text = response.results[0].alternatives[0].transcript``` in STT.py. Not sure why the Google STT is failing to populate the results list now especially since this code worked in previous versions.

image

**Comment by andrew-fennell** [comment]
> index out of range error

I had the seconds being compared to milliseconds... oops! It should be fixed now. I tested with multiple audio files to be safe.

File: CogNative/main.py [hunk 1]
```diff
@@ -9,29 +9,33 @@
...
 if '-sampleAudio' in args and args.index('-sampleAudio') < len(args):
     file_path = Path(args[args.index('-sampleAudio')+1])
 else:
     file_path = Path(input("Enter input audio file path:\n"))
+
+<<<<<<< HEAD
+file_path = "CogNative/examples/TCOC40.wav"
+
+=======
+>>>>>>> aa0274070aa755420fc4033ddf2a3dbeefa6f1b1
```
Comment (JacobSmith1020) [inline CogNative/main.py]: This should not be here.

**Review by JacobSmith1020 — CHANGES_REQUESTED** [review]
Small oversignt. file_path looks like it's being overwritten.

**Review by JacobSmith1020 — APPROVED** [review]
Everything working. Looks good.

**Pull request merged**